Uchu.World/Client/CdClient/ClientCache.cs

```diff
@@ -11,6 +11,141 @@ namespace Uchu.World.Client
     /// </summary>
     public static class ClientCache
```

Comment:
Nice, interested to see the speedups here. Have you checked out the RAM usage? I suspect it's not bad but always good to check.

Reply:
Not quite an A/B comparison since I have this branch and `enhancement/repeatable-missions` ready to go, where `enhancement/repeatable-missions` does not have the changes. All measurements are from logging in, loading Avant Gardens, and doing nothing else.
* Before changes, all logging: 161.2 MB for "World:2003" Process
* Before changes, `error` logging only: 162.6 MB for "World:2003" Process
* After changes, all logging: 264.9 MB for "World:2003" Process *(64.33+)%*
* After changes, `error` logging only: 281.2 MB for "World:2003" Process *(+72.94)%*

This is the first time I actually looked at the numbers, and these numbers really aren't good as Jett's comment that went up before this says.